charactersheet/charactersheet/services/status_service_components/inspiration_component.js

```diff
@@ -22,10 +22,12 @@ function InspirationStatusServiceComponent() {
         var key = CharacterManager.activeCharacter().key();
         var stats = PersistenceService.findFirstBy(OtherStats, 'characterId', key);
 
-        if (!parseInt(stats.inspiration())) {
-            self._removeStatus();
-        } else {
-            self._updateStatus();
+        if (stats) {
```

Comment:
I guess it's not crucial, but I'd prefer a `guard` paradigm here instead of more nestled conditions.

```js
if (!stats) { return; }

if (!parseInt(stats.inspiration())) {
  self._removeStatus();
 } else {
  self._updateStatus();
}
```